packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsConnectionForm.tsx

```diff
@@ -1,47 +1,102 @@
+import styled from '@emotion/styled';
+import { useLingui } from '@lingui/react/macro';
 import { Control, Controller } from 'react-hook-form';
 
 import { Select } from '@/ui/input/components/Select';
 import { TextInput } from '@/ui/input/components/TextInput';
-import styled from '@emotion/styled';
-import { useLingui } from '@lingui/react/macro';
+
 import { H2Title } from 'twenty-ui/display';
 import { Section } from 'twenty-ui/layout';
-import { ConnectionParameters } from '~/generated/graphql';
 
 const StyledFormContainer = styled.form`
   display: flex;
   flex-direction: column;
   gap: ${({ theme }) => theme.spacing(4)};
 `;
 
-type SetttingsAccountsImapConnectionFormProps = {
-  control: Control<ConnectionParameters & { handle: string }>;
+const DEFAULT_IMAP_PORT = 993;
+const DEFAULT_SMTP_PORT = 587;
+
+type ConnectionType = 'IMAP' | 'SMTP';
+
+type ConnectionFormData = {
+  handle: string;
+  host: string;
+  port: number;
+  password: string;
+  secure: boolean;
+};
+
+type SettingsAccountsConnectionFormProps = {
+  control: Control<ConnectionFormData>;
+  connectionType: ConnectionType;
   isEditing: boolean;
-  defaultValues?: Partial<ConnectionParameters & { handle: string }>;
 };
 
-export const SetttingsAccountsImapConnectionForm = ({
+export const SettingsAccountsConnectionForm = ({
   control,
+  connectionType,
   isEditing,
-  defaultValues,
-}: SetttingsAccountsImapConnectionFormProps) => {
+}: SettingsAccountsConnectionFormProps) => {
   const { t } = useLingui();
 
+  const getTitle = () => {
+    return connectionType === 'IMAP'
+      ? t`IMAP Connection Details`
+      : t`SMTP Connection Details`;
+  };
+
+  const getDescription = () => {
```

Comment:
This is not very pretty, let's just put the two forms one below an other as mentioned in the other comment